tests/cases/standalone/common/flow/flow_basic.sql

```diff
@@ -61,6 +61,35 @@ DROP TABLE numbers_input_basic;
 
 DROP TABLE out_num_cnt_basic;
 
+-- test count(*) rewrite
+CREATE TABLE input_basic (
+    number INT,
+    ts TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    PRIMARY KEY(number),
+    TIME INDEX(ts)
+);
+
+CREATE FLOW test_wildcard_basic SiNk TO out_basic AS
+SELECT
+    COUNT(*) as wildcard
+FROM
+    input_basic;
+
+INSERT INTO
+    input_basic
+VALUES
+    (23, "2021-07-01 00:00:01.000"),
+    (24, "2021-07-01 00:00:01.500");
```

Comment:
nitpick: we can add a test to use lowercase as keywords to see if it works, for example:
```
create flow test_wildcard_basic sink to out_basic as
select
    count(*) as wildcard
from
    input_basic;

insert into
    input_basic
values
    (23, "2021-07-01 00:00:01.000"),
    (24, "2021-07-01 00:00:01.500");
```